config.json

```diff
@@ -288,7 +288,13 @@
       "core": false,
       "difficulty": 4,
       "slug": "nth-prime",
-      "topics": null,
+      "topics": [
+      	"arrays",
+      	"exception_handling",
+      	"lists",
+      	"loops",
+      	"integers"
```

Comment:
This looks great! The only tiny problem left is that this should be indented two spaces in from `topics`. Currently it's only one space in :)

Reply:
Okay.

Reply:
I correct the indentation.

Reply:
I'm sorry, but it's still only one space in. The second commit you made has it with two spaces, but then the third commit changed it to only one space again. Do you think you could change it to two spaces? Sorry for being so picky, if you don't want to do it then I can merge it and fix it later :)

Reply:
Did as you said. Please merge this now.

Reply:
Thanks for doing that, I greatly appreciate it :) However, like I've said before, you don't have to tell me to merge things. I will review new changes as soon as I have time and I'll merge them when they're ready :)